api/app/src/main/kotlin/packit/controllers/UserRoleController.kt

```diff
@@ -0,0 +1,22 @@
+package packit.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import packit.model.dto.RolesAndUsersWithPermissionsDto
+import packit.service.UserRoleService
+
+@Controller
+@RequestMapping("/user-role")
```

Comment:
Why is this needed? Having a dedicated endpoint that returns two things sounds like a red flag to me.